releng/org.eclipse.emf.mwe2.language.sdk.dummy/feature.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="org.eclipse.emf.mwe2.language.sdk"
-      version="2.11.2.v20200224-0816"
+      version="2.11.3.v20200224-0816"
```

Comment:
The timestamp looks weird.

Reply:
is random.
will be chaned to real on on release only